test/e2e/ue/get-comment.e2e-spec.ts

```diff
@@ -0,0 +1,192 @@
+    return pactum
+      .spec()
+      .withBearerToken(user.token)
+      .get(`/ue/${ue.code}/comments`)
+      .expectUEComments({
+        items: comments
+          .sort((a, b) =>
+            b.upvotes - a.upvotes == 0
+              ? (<Date>b.createdAt).getTime() - (<Date>a.createdAt).getTime()
+              : b.upvotes - a.upvotes,
+          )
+          .slice(
+            0,
+            Number(
+              app().get(ConfigService).get<number>('PAGINATION_PAGE_SIZE'),
+            ),
+          )
+          .map((comment) => ({
+            ...comment,
+            answers: comment.answers.map((answer) => ({
+              ...answer,
+              createdAt: `${(<Date>answer.createdAt).toISOString()}`,
+              updatedAt: `${(<Date>answer.updatedAt).toISOString()}`,
+            })),
+            updatedAt: `${(<Date>comment.updatedAt).toISOString()}`,
+            createdAt: `${(<Date>comment.createdAt).toISOString()}`,
+          }))
+          .map((comment) => {
+            if (comment.isAnonymous && comment.author.id !== user.id)
+              delete comment.author;
+            return comment;
+          }),
+        itemCount: comments.length,
+        itemsPerPage: Number(
+          app().get(ConfigService).get<number>('PAGINATION_PAGE_SIZE'),
+        ),
+      });
+  });
+
+  it('should return the second page of comments', () => {
+    return pactum
+      .spec()
+      .withBearerToken(user.token)
+      .get(`/ue/${ue.code}/comments`)
+      .withQueryParams('page', 2)
+      .expectUEComments({
+        items: comments
+          .sort((a, b) =>
+            b.upvotes - a.upvotes == 0
+              ? (<Date>b.createdAt).getTime() - (<Date>a.createdAt).getTime()
+              : b.upvotes - a.upvotes,
+          )
+          .slice(
+            Number(
+              app().get(ConfigService).get<number>('PAGINATION_PAGE_SIZE'),
+            ),
+            Number(
+              app().get(ConfigService).get<number>('PAGINATION_PAGE_SIZE'),
+            ) * 2,
+          )
+          .map((comment) => ({
+            ...comment,
+            answers: comment.answers.map((answer) => ({
+              ...answer,
+              createdAt: `${(<Date>answer.createdAt).toISOString()}`,
+              updatedAt: `${(<Date>answer.updatedAt).toISOString()}`,
+            })),
+            updatedAt: `${(<Date>comment.updatedAt).toISOString()}`,
+            createdAt: `${(<Date>comment.createdAt).toISOString()}`,
+          }))
+          .map((comment) => {
+            if (comment.isAnonymous && comment.author.id !== user.id)
+              delete comment.author;
+            return comment;
+          }),
+        itemCount: comments.length,
+        itemsPerPage: Number(
+          app().get(ConfigService).get<number>('PAGINATION_PAGE_SIZE'),
+        ),
+      });
+  });
+
+  it('should return the first page of comments with hidden anonymous authors', () => {
```

Comment:
c'est le même test que le "should return the first page of comments" juste avec l'autre user nan ?